File: pset6/mario.py
block = "#"
space = " "
blocks = 2
height = input("How tall is the half-pyramid? ")
h = int(height)
spaces = h

while ((h < 0) or (h > 23)):
    print("Please enter a positive integer less than 24")
    break

for i in reversed(range(h)):
    for k in range(1, spaces):
        print(space, end="")
        
    for j in range(0, blocks):
        print(block, end="")
        
    blocks += 1
    spaces -= 1
    print(" ")

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    printf("How tall is the half-pyramid? ");
   // int height = get_int();
    do 
    {
        height = get_int();
    }
    while (height > 23 || height < 0);
    char block[] = "#";
    char space[] = " ";
    int blocks = 1;
    int spaces = height;

    for (int i = height; i > 0; i--){ // levels of pyramid
 
        for (int k = 1; k < spaces; k++){ // spaces per level
             printf("%c", *space);
        }
 
        for (int j = 0; j < blocks + 1; j++){ // blocks per level
             printf("%c", *block);
        }

        blocks++;
        spaces--;
        printf("\n");
    }

}